contracts/UFragmentsPolicy.sol

```diff
@@ -264,6 +264,40 @@ contract UFragmentsPolicy is Ownable {
         );
     }
 
+    /**
+     * @return Computes the percentage of supply to be added or removed.
+     * Using the function in https://github.com/ampleforth/AIPs/blob/master/AIPs/aip-5.md
+     * @param normalizedRate a DECIMALS decimal fixed point number.
```

Comment:
I think `normalizedRate` needs an explanation. It could be in the @dev block to keep the @param line short.

Reply:
added rate/target in the param